src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ExtensionComponent.scala

```diff
@@ -0,0 +1,70 @@
+package org.broadinstitute.dsde.workbench.leonardo.db
+
+import org.broadinstitute.dsde.workbench.leonardo.model.UserJupyterExtensionConfig
+
+case class ExtensionRecord(clusterId: Long, extensionType: String, name: String, value: String)
+
+trait ExtensionComponent extends LeoComponent {
+  this: ClusterComponent =>
+
+  import profile.api._
+
+  class ExtensionTable(tag: Tag) extends Table[ExtensionRecord](tag, "NOTEBOOK_EXTENSIONS") {
+    def clusterId = column[Long]("clusterId")
+
+    def extensionType = column[String]("extensionType", O.Length(254))
+
+    def name = column[String]("name", O.Length(254))
+
+    def path = column[String]("path", O.Length(254))
```

Comment:
Since this is an arbitrary gs:// path, we might want to bump the length to 1024. The CLUSTER table does this as well: https://github.com/DataBiosphere/leonardo/blob/develop/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala#L69-L70